llvmlite/binding/newpassmanagers.py

```diff
@@ -227,6 +236,39 @@ def getFunctionPassManager(self):
                 self, self._pto.speed_level, self._pto.size_level)
         )
 
+    def start_pass_timing(self):
+        """Enable the pass timers.
+
+        Raises
+        ------
+        RuntimeError
+            If pass timing is already enabled.
+        """
+        if self._time_passes_handler:
+            raise RuntimeError("Pass builder should only have one \
+                pass timer at a time")
```

Comment:
I think this is a little clearer (the current message implies there's a way to remove a pass timer, which I think there isn't):

```suggestion
            raise RuntimeError("Pass timing can only be done once")
```